components/Componenten/form-elements/group-filter.config.yml

```diff
@@ -0,0 +1,12 @@
+status: ready
+collated: true
+context:
```

Comment:
Extra varianten toevoegen:
* Met info
* Met info open
* Met error
* Met helptekst

En ook toevoegen in het formulier, checken hoe het lijnt in horizontal als vertical